dom-element.js

```diff
@@ -97,13 +97,13 @@ DOMElement.prototype.setAttributeNS =
     function _Element_setAttributeNS(namespace, name, value) {
         var colonPosition = name.indexOf(":")
         var localName = colonPosition > -1 ? name.substr(colonPosition + 1) : name
-        var attributes = this._attributes[namespace] || (this._attributes[namespace] = {})
-        attributes[localName] = value
+        var attributes = namespace === null ? this : (this._attributes[namespace] || (this._attributes[namespace] = {}))
```

Comment:
We cannot write to `this`. We must write to `this._attributes['default-namespace']` or something. We can even have `this._attributes` and `this._nsAttributes`